### PR TITLE
Dashboard layout

### PR DESCRIPTION
**Pull request by eserpin**
I've finished the basic layout of the dashboard with no styling. The necessary displayed information is all there except for the cumulative dive time, and the functionality is pretty much all there, other than a few additions I'd like to make such as highlighting selected tags. 
The majority of the work left to do on the dashboard is implementation of the api routes, styling/positioning(which I suck at), as well as a possible implementation of a graph?

This is a pretty big pull request so I'd be happy to go over the changes with whoever decides to review it, as well as show what the dashboard page actually visually looks like at the moment. Feedback would also be appreciated!.

### CODE REVIEW

**Review by kennyctran — CHANGES_REQUESTED** [review]
I haven't taken a look at the code, but the Builds and deployment failed so something is breaking within the code. I'd go back and build in dev environment to narrow down the scope of what went wrong.

**Review by kennyctran — COMMENTED** [review]
DashboardLogDisplay -- I think it might be best to abstract this into component files. It's very hard to parse through what's actually happening here, which might present larger issues in the coming days. I recommend pulling all these renders into their own components or at least abstracting the unique parts of each render into their own components.

dashBoardTagList-- Change file to fit naming convention DashboardTagList
Lines 25 and 27 are identical even though they're in an if-else block

**Comment by eserpin** [comment]
@kennyctran 
I made the requested changes except the if else statement in DashboardTagList. All I wanna do is return a different colored Chip component in one of them, but I haven't been able to because of my inexperience with Material UI. I'm gonna need to do some reading on how Material UI themes and styling works.

**Review by kennyctran — COMMENTED** [review]
Create a page for dashboard

**Review by kennyctran — COMMENTED** [review]
For line 26 and 28 for the Chip

<Chip color={(conditional statement) ? 'primary' : 'secondary' } />

**Comment by eserpin** [comment]
Added dashboard page and implemented color changing Chips in tag list component

**Review by kennyctran — APPROVED** [review]
Looks dope!!

Can you create two Trello tickets? One for changing the colors being used and another for handling duplicate tags (sea turtles shows twice right now)
